Source/Scene/Model.js

```diff
@@ -4438,7 +4437,7 @@ define([
                     ModelUtility.updateForwardAxis(this);
 
                     // glTF pipeline updates, not needed if loading from cache
-                    if (!this._loadRendererResourcesFromCache) {
+                    if (!defined(this.gltf.extras.sourceVersion)) {
```

Comment:
Was this the lighting issue fix? Are you using `sourceVersion` as a way to know if this `gltf` object has already gone through this code-path so it doesn't go through when it's loaded in other viewers?

Reply:
Yeah, exactly.